src/group_theory/subsemigroup/centralizer.lean

```diff
@@ -100,10 +100,12 @@ end
 lemma centralizer_subset [has_mul M] (h : S ⊆ T) : centralizer T ⊆ centralizer S :=
 λ t ht s hs, ht s (h hs)
 
+@[to_additive add_center_subset_add_centralizer]
```

Comment:
What name does `to_additive` generate here? Why does it fail here and doesn't fail at `center_le_centralizer`?

Reply:
> and generates a correct name for `center_le_centralizer`?

Because the correct additivized name for `subgroup.center_le_centralizer` is `add_subsemigroup.center_le_centralizer`, there's no need to replace `centralizer` with `add_centralizer`.

Only the `set` lemmas need these renames.